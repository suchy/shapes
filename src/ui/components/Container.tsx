import { EDEADLK } from 'constants';
import styled from 'styled-components';

export const Container = styled.div`
  width: 100%;
  margin: 0 auto;
  padding: 0 3rem;
  max-width: 144rem;
`;
