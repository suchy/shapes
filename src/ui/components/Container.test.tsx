import { render } from '@testing-library/react';
import { Container } from './Container';

describe('Container', () => {
  it('should match snapshot', () => {
    const { container } = render(<Container />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
