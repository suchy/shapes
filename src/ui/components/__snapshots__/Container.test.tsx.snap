// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should match snapshot 1`] = `
<div
  class="sc-bdfBwQ gotimp"
/>
`;
